reference/reference-styling.html

```diff
@@ -113,7 +118,7 @@ <h2>Text properties: always forward-compatible</h2>
         </div>
 
         <h2>Size</h2>
-        <p>Cell size can currently be changed simply by setting <code>width</code> and <code>height</code> on a cell. If the cell contains text, you may also want to change the <code>font-size</code> property, as well as for small text.</p>
+        <p>Cell size can currently be changed simply by setting the CSS variable <code>--cell-size</code> on a puzzle. If the cells contain text, you may also want to change the <code>font-size</code> property, as well as for small text.</p>
```

Comment:
nit: 

"If the cells contain text, you may also want to change the <code>font-size</code> property to increase or decrease the font size."

Reply:
Going to skip this since a) it's already in the example and b) the default 0.8 scaling is good.

Reply:
Oh wait, misread sorry. I'll put it in, thanks.